src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -1620,6 +1620,121 @@ public void Method()
             await VerifyItemExistsAsync(markup, "nullables");
         }
 
+        [WorkItem(1220195, "https://github.com/dotnet/roslyn/issues/23590")]
```

Comment:
@jasonmalinowski The work item link here doesn't seem correct.

Reply:
Yeah, I totally didn't copy/paste that from the other test in the file and forget to update it. Totally.

:wink:

Fix here: https://github.com/dotnet/roslyn/pull/50246. Good catch!